documentation/src/test/java/example/testinterface/TestInterfaceDemoTests.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package example.testinterface;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+//tag::user_guide[]
+public class TestInterfaceDemoTests implements TestConsoleLogger, TimeExecutionLogger, DynamicTests {
```

Comment:
By convention, all examples in the User Guide are called `*Demo`. So please rename your test class accordingly.